types.ts

```diff
@@ -839,11 +846,9 @@ export interface VariousAuthentication {
 }
 
 /**
- * The union of supported authentication methods.
+ * The union of authentication types that pack makers are allowed to use.
```

Comment:
Nit, this comment is confusing, intuitively I think people would expect everything in a public SDK is "usable" by makers. I think the implication here is that VariousAuthentication is only allowed for 1P packs at present?

Reply:
Done!